internal/openshift/utils.go

```diff
@@ -15,8 +23,48 @@ var (
 		Kind:    "ConsolePlugin",
 	}
 	ConsolePluginsResource = consolev1.SchemeGroupVersion.WithResource("consoleplugins")
+
+	ClusterVersionGroupKind = schema.GroupKind{
+		Group: "config.openshift.io",
+		Kind:  "ClusterVersion",
+	}
+	ClusterVersionResource = configv1.SchemeGroupVersion.WithResource("clusterversions")
+
+	ConsolePluginImageURL    = env.GetString("RELATED_IMAGE_CONSOLEPLUGIN", "quay.io/kuadrant/console-plugin:latest")
```

Comment:
I'm kind of iffy about this now that I've had some time to consider. I think we'd be better of with a more flexible map and some way to match with a range. Needs a bit more thought I think

Reply:
Maybe we could have a map with some contraints instead?

Something like:

```
RELATED_IMAGE_CONSOLEPLUGIN_MAP='[{"constraint":"< 4.19","image":"v0.1.x"},{"constraint":">= 4.19","image":"v0.2.x"}]'
```